docs/notebooks/deep_mcdropout.py

```diff
@@ -46,61 +52,278 @@ def objective(x, error=True):
 )
 from trieste.models.optimizer import KerasOptimizer
 
-def build_cubic_model(data:Dataset, dropout:str="standard") -> MCDropout:
-    num_hidden_layers = 3
-    num_nodes = 50
-    activation = "relu"
-    rate = 0.2
-
-    dropout_nn = build_vanilla_keras_mcdropout(
-        data, 
-        num_hidden_layers=num_hidden_layers,
-        units=num_nodes,
-        activation=activation,
-        rate=rate,
-        dropout=dropout
+
+# def build_cubic_model(data: Dataset) -> MCDropout:
```

Comment:
Yes, the cubic model seemed reasonable and fine. The notebook is a bit of a mess because I was more merging it in for David to pull and then work on.